recipes/waves/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "waves" %}
+{% set version = "0.6.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lanl/waves/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 62fbb8642b79b39cf369ced0ee7c2638caed4b20b9a9e8a9c4afb42e3f4edecc
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script:
+    - {{ PYTHON }} setup.py install
+    - scons man html-github
+    - {{ PYTHON }} package_documentation.py
+  ignore_run_exports:
+    - python_abi
+
+requirements:
+  build:
+    - python {{ python }}
+    - git
```

Comment:
Is `git` needed if you're setting `SETUPTOOLS_SCM_PRETEND_VERSION`?

Reply:
Probably not. Thanks for the catch.